archetypes/Portfolio/ClaimedTokensTable/ClaimedTokenRow.tsx

```diff
@@ -51,25 +53,39 @@ export const ClaimedTokenRow: React.FC<ClaimedTokenRowProps & ClaimedRowActions>
                 <div>{`${leveragedNotionalValue.toFixed(3)} ${settlementTokenSymbol}`}</div>
             </OverviewTableRowCell>
             <ActionsCell>
-                <ActionsButton
-                    size="xs"
-                    variant="primary-light"
-                    // will never be disabled if it gets included as a row it will always be either to stake or to unstake
-                    onClick={() => onClickStake(address, shouldStake ? 'stake' : 'unstake')}
-                >
-                    {shouldStake ? 'Stake' : 'Unstake'}
-                </ActionsButton>
-                <ActionsButton
-                    size="xs"
-                    variant="primary-light"
-                    disabled={!balance.toNumber()}
-                    onClick={() => onClickCommitAction(poolAddress, side, CommitActionEnum.burn)}
-                >
-                    Burn
-                </ActionsButton>
+                <PortfolioStakeTooltip>
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        // will never be disabled if it gets included as a row it will always be either to stake or to unstake
+                        onClick={() => onClickStake(address, shouldStake ? 'stake' : 'unstake')}
+                    >
+                        {shouldStake ? 'Stake' : 'Unstake'}
+                    </ActionsButton>
+                </PortfolioStakeTooltip>
+                <PortfolioSellTooltip>
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        disabled={!balance.toNumber()}
+                        onClick={() => open(constructBalancerLink(address, NETWORKS.ARBITRUM, true), '_blank')}
+                    >
+                        Sell
+                    </ActionsButton>
+                </PortfolioSellTooltip>
+                <PortfolioBurnTooltip>
+                    <ActionsButton
+                        size="xs"
+                        variant="primary-light"
+                        disabled={!balance.toNumber()}
+                        onClick={() => onClickCommitAction(poolAddress, side, CommitActionEnum.burn)}
+                    >
+                        Burn
+                    </ActionsButton>
+                </PortfolioBurnTooltip>
```

Comment:
this currently renders the placeholder for {time}, it needs to pass the actual expected time in

Reply:
Resolved